radarr/radarr.go

```diff
@@ -431,3 +431,19 @@ func (r *Radarr) SendCommand(cmd *CommandRequest) (*CommandResponse, error) {
 
 	return &output, nil
 }
+
+// MarkHistoryItemAsFailed marks the given history item as failed by id.
+func (r *Radarr) Fail(historyID int64) error {
+	if historyID < 1 {
+		return fmt.Errorf("invalid history ID: %d", historyID)
+	}
+
+	params := make(url.Values)
+
+	_, err := r.Post("v3/history/failed/"+strconv.FormatInt(historyID, starr.Base10), params, nil)
```

Comment:
Is this supposed to be a Post? The body is `nil`, and that seems suspect.
If it's supposed to a Post please fix the `Errorf` line below that says `api.Get` (it's a Post not a Get).

Same for Sonarr changes.